src/target/stm32f1.c

```diff
@@ -357,17 +357,25 @@ static int stm32f1_flash_unlock(target *t, uint32_t bank_offset)
 	return 0;
 }
 
-static bool stm32f1_flash_busy_wait(target *t, uint32_t bank_offset)
+static void stm32f1_flash_clear_eop(target *t, uint32_t bank_offset)
+{
+	uint32_t sr = target_mem_read32(t, FLASH_SR + bank_offset);
```

Comment:
`sr` here should be named `status`, which avoids the clang-tidy warning about too short a variable name and lends more clarity